features/distributed/kv-store/cassandra/src/main/resources/cql/kv.cql

```diff
@@ -0,0 +1,11 @@
+
+CREATE KEYSPACE $KEYSPACE$
+    WITH replication = {'class': 'SimpleStrategy', 'replication_factor': $REPLICATION_FACTOR$};
+
+CREATE TABLE $KEYSPACE$.kvstore (
+  key varchar,
+  context varchar,
+  value blob,
+  lastUpdated timestamp,
+  PRIMARY KEY ((context, key))
```

Comment:
Can we change this to `((context), key)` so that it's possible to enumerate all of the keys within a specific context?

Reply:
I was considering that. It does allow use to enumerate by context as you mentioned. However the downside is that all keys for a given context will have the same primary Cassandra node instead of being evenly distributed throughout the cluster. For now I will leave it as is since we don't currently have the use case of enumerating a context.